selfdrive/car/toyota/carcontroller.py

```diff
@@ -49,15 +61,15 @@ def update(self, enabled, CS, frame, actuators, pcm_cancel_cmd, hud_alert,
     # *** compute control surfaces ***
 
     # gas and brake
+    apply_gas = 0.
+    apply_accel = actuators.gas - actuators.brake
 
-    apply_gas = clip(actuators.gas, 0., 1.)
-
-    if CS.CP.enableGasInterceptor:
-      # send only negative accel if interceptor is detected. otherwise, send the regular value
+    if CS.CP.enableGasInterceptor and enabled and CS.out.vEgo < MIN_ACC_SPEED and \
+            apply_accel * CarControllerParams.ACCEL_SCALE > coast_accel(CS.out.vEgo):
       # +0.06 offset to reduce ABS pump usage when OP is engaged
-      apply_accel = 0.06 - actuators.brake
-    else:
-      apply_accel = actuators.gas - actuators.brake
+      # converts desired acceleration to gas percentage for pedal
+      apply_gas = clip(compute_gb_pedal(apply_accel * CarControllerParams.ACCEL_SCALE, CS.out.vEgo), 0., 1.)
+      apply_accel = 0.06
```

Comment:
Why not 0?

Reply:
And then put the next two lines in the else?

Reply:
The hysteresis and clipping lines in the else? And I was confused about the original reason it was added for offsetting the brake, you think we don't need it?

The only thing is, if the function wants to apply 0% gas to coast up at a low speed and desired accel is 0, the car might brake since the coast accel at that low speed (0-2.5m/s) is positive.

Reply:
Can you check if it runs the pump when you send exactly 0? If not then we should just use that.

Reply:
I think it changes based on speed, but will do! For example if we're at less than 1 or 2 mph, but still moving, sending 0 accel will make the car brake to a stop for some reason. I'll make sure though. Also how does the new change look? I thought we should be sending the function what a TSS2 Toyota would get, the accel after it goes through the hysteresis and clipping steps.

Reply:
At some speed the PCM will decide that's time to stop and will just brake no matter what you do (unless you send very high accel). So 0 or 0.06 is probably not going to make a difference for that.

Reply:
New change looks good. Think in this case we just want to make sure it sends zero, otherwise the hysteresis could send some non zero value and you could have gas/brake fighting.